dev-embedded/stm32flash/Manifest

```diff
@@ -1 +1,2 @@
 DIST stm32flash-0.5.tar.gz 41087 BLAKE2B 581ef883537783fafcff8a7471a697f50929b572d847791d692799040fd54845a67a7b6b608401df24f05810ae81b7e19df646f2835bc3d7cd35888d8649be6a SHA512 a01875a315d92d5e347372b736f2afa2f633851bf13af1326341c0c91e17cef54b6730ed1279280e86d3797033c856dfc25e8c595e010772b9287aaf965273f9
+DIST stm32flash-0.6.tar.gz 376186 SHA512 2d92ecf4f82b8a702b596cbf3c4ff594f67083f7dcec82c064555f73b47834b41a918efa1846e47fc8740ae63fa818b16dfb5b38fa242c17f465666a651eee3a
```

Comment:
We still use both SHA512 and BLAKE2B to verify hashes. You should be able to generate this via either
`repoman manifest`, 
`pkgdev commit`, or
`ebuild stm32flash-0.6.ebuild manifest` when positioned in ::gentoo repo dir.